Gems/Atom/RPI/Code/Source/RPI.Public/View.cpp

```diff
@@ -125,6 +125,33 @@ namespace AZ
             m_drawListContext.AddDrawItem(drawListTag, drawItemProperties);
         }
 
+        void View::ApplyFlags(uint32_t flags)
```

Comment:
Definitely worth documenting how these flags are used I think - without seeing this PR or searching to eventually find the various "caster moved" bits it's not easy to see 😉

Reply:
yeah, keeping things really decoupled can be a two-edged sword. Makes it really easy for other system to inject their own bits for whatever they might want to track, but it also makes it harder for someone reading the code to figure out how it all fits together.